static/js/lib/url.js

```diff
@@ -52,6 +52,11 @@ export const embedlyThumbnail = (
     url
   )}&height=${height}&width=${width}&grow=true&errorurl=${blankThumbnailUrl()}`
 
+export const embedlyImage = (key: string, url: string, height: number) =>
```

Comment:
what do you think about calling this `embedlyResizeURL`? just something to make it a little more clear exactly what it's doing

Reply:
Renamed to `embedlyResizeImage`